index.ios.js

```diff
@@ -138,6 +138,14 @@ export default class NotificationsIOS {
   }
 
   /**
+   * See what push permissions are currently enabled. 
```

Comment:
According to https://travis-ci.org/wix/react-native-notifications/builds/196439400#L477 you need to remove the trailing space here. If you do that, maybe this pull-request can be merged.